src/applications/vaos/reducers/newAppointment.js

```diff
@@ -650,15 +650,10 @@ export default function formReducer(state = initialState, action) {
 
       if (state.pastAppointments) {
         const pastAppointmentDateMap = new Map();
-        const rootOrgId = getIdOfRootOrganization(
-          state.parentFacilities,
-          state.data.vaParent,
-        );
-
         const org = state.parentFacilities.find(
           parent => parent.id === state.data.vaParent,
         );
-        const siteId = getSiteIdFromOrganization(org);
+        const siteId = getSiteIdFromOrganization(org).substring(0, 3);
```

Comment:
The old call in some cases would return 5 digit codes, this should always be 3 digits.